styles/type.scss

```diff
@@ -43,7 +43,6 @@ $p-xl-line-height: 3.6rem;
 
   margin: 0 2px;
   font-size: 0.75rem;
-  letter-spacing: 0.4px;
 
```

Comment:
@mokaymokay restored letter-spacing for caption description!

Also changed .tiny font size from .9375rem(15px) to .9rem(14.4rem) for consistency(all the other small types in the page use .9rem, so). 
I changed the type system in the past to apply [the final design](https://www.figma.com/file/M6zHKnDZ7ALV5pZxPvVSHZ/sanctuary.computer-refresh?node-id=84%3A461&t=V98XamQZH8N09C4b-1) but maybe the design didn't consider type change? 

But I still think keeping the slightly larger font size (10px to 14.4px) for .tiny texts(ex. caption texts) is better for readability, accessibility, rather than reverting all type size changes. What do you think?

Reply:
> changed .tiny font size from .9375rem(15px) to .9rem(14.4rem) for consistency(all the other small types in the page use .9rem, so)
Yeah I like 0.9rem as well.

The designs have links at 1.25rem on desktop and 0.75rem on mobile (sigh), but they seem to be low-fidelity so can we check with Hugh and Elie if they are OK with our decision to override the designs?

Reply:
> big(ugly)

xD